coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorGrpcService.java

```diff
@@ -193,6 +194,42 @@ public void appHeartbeat(
     responseObserver.onCompleted();
   }
 
+  @Override
+  public void accessCluster(AccessClusterRequest request, StreamObserver<AccessClusterResponse> responseObserver) {
+    String accessInfo = request.getAccessInfo();
+    StatusCode statusCode = StatusCode.SUCCESS;
+    AccessClusterResponse response;
+    AccessManager accessManager = coordinatorServer.getAccessManager();
+
+    if (accessManager == null) {
+      LOG.warn("AccessManager is disabled return success directly for {}", accessInfo);
+      response = AccessClusterResponse
+          .newBuilder()
+          .setStatus(statusCode)
+          .setRetMsg("No AccessManager")
+          .build();
+    } else {
+      AccessCheckResult result = accessManager.handleAccessRequest(accessInfo);
+      if (!result.isSuccess()) {
+        statusCode = StatusCode.INTERNAL_ERROR;
```

Comment:
It's not internal error, just rss reject because of accessible check

Reply:
Add an ACCESS_FAIL?